db/migrate/20230126233821_delete_calculator_currency_from_spree_preferences.rb

```diff
@@ -0,0 +1,15 @@
+class DeleteCalculatorCurrencyFromSpreePreferences < ActiveRecord::Migration[6.1]
+  class SpreePreference < ActiveRecord::Base
+  end
+
+  def up
+    # Delete all currency preferences for calculators, they are no longer used.
+    rows = SpreePreference.where("key like '/calculator%/currency/%'").delete_all
+    puts "#{rows} rows deleted."
```

Comment:
This query is so simple and so minimal on Rails use that I would have `execute`d plain SQL. But this is good, too. I think it's just a recent preference of mine after years of trying to *railsify* everything I'm now trying to reduce everything to the bare minimum. And I think that `execute` would have given you the output of rows deleted out of the box.

Reply:
Good point, using ActiveRecord doesn't give any advantage here, although perhaps helps avoid errors in writing SQL directly. 
I can see that's what [Ana did recently](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/db/migrate/20230130032659_delete_ssl_preferences.rb) too.